persist/fs/seek.go

```diff
@@ -129,8 +146,9 @@ func newSeeker(opts seekerOpts) fileSetSeeker {
 		filePathPrefix:             opts.filePathPrefix,
 		infoFdWithDigest:           digest.NewFdWithDigestReader(opts.infoBufferSize),
 		indexFdWithDigest:          digest.NewFdWithDigestReader(opts.dataBufferSize),
-		dataReader:                 bufio.NewReaderSize(nil, opts.seekBufferSize),
 		digestFdWithDigestContents: digest.NewFdWithDigestContentsReader(opts.infoBufferSize),
+		bloomFilterFdWithDigest:    digest.NewFdWithDigestContentsReader(opts.dataBufferSize),
```

Comment:
Here too, think you want `NewFdWithDigestReader` instead.

Reply:
will fix